Dockerfile

```diff
@@ -17,19 +17,32 @@
 FROM codait/max-base:v1.1.3
 
 # Fill in these with a link to the bucket containing the model and the model file name
-ARG model_bucket=https://s3.us-south.cloud-object-storage.appdomain.cloud/max-assets-prod/max-named-entity-tagger/1.0.0
+ARG model_bucket=https://s3.us-south.cloud-object-storage.appdomain.cloud/max-assets-dev/max-named-entity-tagger/1.1.0
 ARG model_file=assets.tar.gz
 
-WORKDIR /workspace
+ARG use_pre_trained_model=true
 
-RUN wget -nv --show-progress --progress=bar:force:noscroll ${model_bucket}/${model_file} --output-document=assets/${model_file} && \
-  tar -x -C assets/ -f assets/${model_file} -v && rm assets/${model_file}
+RUN if [ "$use_pre_trained_model" = "true" ] ; then\
+     # download pre-trained model artifacts from Cloud Object Storage
+     wget -nv --show-progress --progress=bar:force:noscroll ${model_bucket}/${model_file} --output-document=assets/${model_file} &&\
+     tar -x -C assets/ -f assets/${model_file} -v && rm assets/${model_file} ; \
+    fi
 
 COPY requirements.txt /workspace
 RUN pip install -r requirements.txt
 
 COPY . /workspace
 
+RUN if [ "$use_pre_trained_model" = "true" ] ; then \
+      # validate downloaded pre-trained model assets
+      md5sum -c md5sums.txt ; \
```

Comment:
I think we've been using SHA512 sums for the latest version of MAX models.

Reply:
I'll make a separate PR for that, thanks